packages/core/src/constants/type/QueryParams.ts

```diff
@@ -11,4 +12,7 @@ export interface QueryParams {
    */
   chainId?: ChainId
   isStatic?: boolean
+  fromBlock?: BlockTag
+  toBlock?: BlockTag
```

Comment:
The `fromBlock` and `toBlock` do not make sense for all other hooks, like `useEtherBalance` for example.
You could maybe imagine it would return a list of balances across those blocks but it's not currently implemented.
Same with `blockHash` - it could return balance for that block, but it's not implemented and can lead to unexpected behaviour if the user uses those fields for queries other than `useLogs()`.
So I propose to leave `QueryParams` intact, and instead add:
```
/**
 * If both `blockHash` and `fromBlock/toBlock` are defined, `blockHash` takes precedence.
 */
interface LogsQueryParams extends QueryParams {
  fromBlock?: BlockTag
  toBlock?: BlockTag
  blockHash?: string
}
```

Reply:
Great suggestion. I created LogQueryParams and subsequently exported it along with QueryParams which should probably also be exported.